sql/2025/fonts/design/fonts_designer.sql

```diff
@@ -0,0 +1,54 @@
+-- Section: Design
+-- Question: Which designers are popular?
+-- Normalization: Pages
+
+-- INCLUDE https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/{year}/fonts/common.sql
+
+WITH
+designers AS (
+  SELECT
+    client,
+    NULLIF(TRIM(STRING(JSON_QUERY(payload, '$._font_details.names[9]'))), '') AS designer,
+    COUNT(DISTINCT page) AS count,
+    ROW_NUMBER() OVER (PARTITION BY client ORDER BY COUNT(DISTINCT page) DESC) AS rank
+  FROM
+    `httparchive.crawl.requests`
+  WHERE
+    date = @date AND
+    type = 'font' AND
+    is_root_page AND
```

Comment:
If we wanted to get greater coverage you could drop `is_root_page` and change:
```sql
COUNT(DISTINCT page) AS count
```
to
```sql
COUNT(DISTINCT root_page) AS count
```

Reply:
The query is specifically about root pages. Last year, we decided it would be confusing otherwise. I remember you explained that it could be one additional page or several, but that would be a strange selection, and it would also be more difficult to explain and motivate in the chapter.

This pattern repeat is several queries if I remember right.

Reply:
Yeah it depends if you want to measure "home page usage" or "site usage". As often home pages aren't totally representative of site usage.

I mean it's only 2 pages for a site (home page and one other page) so even then it's not a complete representation of the whole site anyway. But it could be _more_ representative than just the home page if the home page is a special page (as it often is).

But maybe one for next year. Also maybe less relevant for fonts since home page likely has the brand fonts anyway.